components/d2l-filter-dropdown/d2l-filter-dropdown-page.js

```diff
@@ -22,7 +22,7 @@ class D2LFilterDropdownPage extends mixinBehaviors([D2L.PolymerBehaviors.FilterD
 			<d2l-menu label="[[parentTitle]]">
 				<dom-repeat items="[[options]]" as="o">
 					<template>
-					<d2l-menu-item-checkbox text="[[o.title]]" value="[[o.key]]" hidden$="[[!o.display]]" selected=[[o.selected]]></d2l-menu-item-checkbox>
+					<d2l-menu-item-checkbox text="[[o.title]]" value="[[o.key]]" selected=[[o.selected]]></d2l-menu-item-checkbox>
```

Comment:
As far as I can tell, `display` was only used by the search stuff so I removed it

Reply:
Actually, isn't this needed in order to keep the total counts correct?  To still say you've selected 3 filters, even if you've searched those three away?

Reply:
- [ ] make sure the total counts are correct

Reply:
Ah this exposed a whole range of nasty bugs (we were really relying on having the entire list of options)